rskj-core/src/main/java/co/rsk/peg/bitcoin/UtxoUtils.java

```diff
@@ -0,0 +1,78 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.VarInt;
+import com.google.common.primitives.Bytes;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.spongycastle.util.encoders.Hex;
+
+public class UtxoUtils {
+    private UtxoUtils() {}
+
+    /**
+     * Decode a {@code byte[]} of encoded outpoint values.
+     *
+     * @param encodedOutpointValues
+     * @return {@code List<Coin>} the list of outpoint values decoded in the same ordered they were before encoding.
+     * Or an {@code Collections.EMPTY_LIST} when {@code encodedOutpointValues} is {@code null} or {@code empty byte[]}.
+     */
+    public static List<Coin> decodeOutpointValues(byte[] encodedOutpointValues){
+        if (encodedOutpointValues == null || encodedOutpointValues.length == 0) {
+            return Collections.emptyList();
+        }
+        int offset = 0;
+        List<Coin> outpointValues = new ArrayList<>();
+
+        while (encodedOutpointValues.length > offset){
```

Comment:
There is a bunch of places where you are not adding a space before the bracket. If possible, make sure we are formatting this right.